src/pages/BorrowerProfile/FundedBorrowerProfile.vue

```diff
@@ -111,6 +111,14 @@ export default {
 	metaInfo() {
 		const canonicalUrl = `https://${this.$appConfig.host}${this.$route.path}`.replace('funded', 'lend');
 		return {
+			title: `${this.loan.name ?? ''} From ${this.loan.geocode.country.name ?? ''}'s has been funded`,
```

Comment:
`this.loan` is always defined for this component?

Reply:
@dyersituations It is defined as an empty object.

![FundedBorrowerProfile vue — kiva 2022-06-06 16-20-49](https://user-images.githubusercontent.com/102815442/172251348-99f4a544-7ae2-4df0-a9cd-f284e953023b.png)

Reply:
So, if I remove **_this_** it breaks because the undefined properties

![FundedBorrowerProfile vue — kiva 2022-06-06 16-23-54](https://user-images.githubusercontent.com/102815442/172251919-554afbf3-26c5-4731-8a5f-07051d3938de.png)

Reply:
In that case, you probably want `this.loan.geocode?.country?.name`? Since by default `loan` is an empty object.